<!-- frobit_plot example launch file -->

<launch>
	<group ns="fmProcessors">
		<node pkg="frobit_plot" name="frobit_plot" type="frobit_plot_node.py" output="screen" >
			<param name="pose_sub" value="/fmKnowledge/pose" />
			<param name="gnss_sub" value="/fmInformation/gpgga_tranmerc" />
			<param name="odom_sub" value="/fmKnowledge/odometry" />
			<param name="wptnav_status_sub" value="/fmInformation/wptnav_status" />
			<param name="plot_pose_track" value="true" />
			<param name="plot_gnss_track" value="false" />
			<param name="plot_odometry_track" value="false" />
			<param name="plot_pose_yaw" value="false" />
			<param name="map_trackpoint_threshold" value="0.05" />
			<param name="map_max_trackpoints" value="1000" />
			<param name="map_minimum_size" value="5.0" />
			<param name="map_easting_offset" value="0.0" />
			<param name="map_northing_offset" value="0.0" />
			<param name="map_update_frequency" value="5" />
			<param name="map_title" value="Frobit track" />
			<param name="map_window_size" value="5.0" />
			<param name="avatar_extension_front" value="0.29" />
			<param name="avatar_extension_rear" value="0.06" />
			<param name="avatar_extension_lateral" value="0.13" />
			<param name="avatar_reverse" value="false" />
		</node>
	</group>
</launch>

